broker/submissions/routes.py

```diff
@@ -16,23 +17,47 @@
 )
 
 
+@submissions_bp.route('/<submission_uuid>/spreadsheet', methods=['POST'])
+def generate_spreadsheet(submission_uuid):
+    submission = app.ingest_api.get_submission_by_uuid(submission_uuid)
+    spreadsheet_job = submission.get('lastSpreadsheetDownloadJob', {}) or {}
+
+    message = 'The spreadsheet is being generated.'
+    if not spreadsheet_job.get('createdDate') or (spreadsheet_job.get('createdDate') and spreadsheet_job.get('finishedDate')):
+        spreadsheet_export_service = ExportToSpreadsheetService(app.ingest_api)
+        spreadsheet_export_service.async_export_and_save(submission_uuid, app.SPREADSHEET_STORAGE_DIR)
+        return response_json(HTTPStatus.ACCEPTED, {'message': message})
+    else:
+        return response_json(HTTPStatus.ACCEPTED, {'message': message})
```

Comment:
would that be the case? i.e. if we do not have a `lastSpreadsheetDownloadJob`, could that mean that one wasn't requested yet?

Reply:
yup, in that case, it will meet the condition above :)